client/src/extension.ts

```diff
@@ -47,16 +48,35 @@ export async function activate(context: ExtensionContext) {
 		}
 	};
 
-	const documentSelector = [
-		{language: 'objectscript'},
-		{language: 'objectscript-class'},
-		{language: 'objectscript-csp'},
-		{language: 'objectscript-macros'}
+	// The languages we handle
+	const targetLanguages = [
+		'objectscript',
+		'objectscript-class',
+		'objectscript-csp',
+		'objectscript-macros',
 	];
 
+	// The uri schemes we handle those languages for
+	const targetSchemes = [
+		'isfs',
+		'isfs-readonly',
+		'objectscripts',

```

Comment:
This is a typo. The scheme is 'objectscript'. Also, do we need to include the 'objectscriptxml' schema from the main extension?

Reply:
Well spotted re the typo. And I've added `objectscriptxml` as it won't hurt to have it there.